src/v2/computations-and-formatting.ts

```diff
@@ -42,7 +42,29 @@ function getEthAndUsdBalance(
   return [balanceInEth.toString(), balanceInUsd];
 }
 
-function computeUserReserveData(
+/*
+type ComputeUserReserveDataPoolReserve = Pick<
+  ReserveData,
+  | 'price'
+  | 'decimals'
+  | 'liquidityIndex'
+  | 'liquidityRate'
+  | 'lastUpdateTimestamp'
+  | 'variableBorrowIndex'
+  | 'variableBorrowRate'
+>;
+
+type ComputeUserReserveDataUserReserve = Pick<
+  UserReserveData,
+  | 'scaledATokenBalance'
+  | 'scaledVariableDebt'
+  | 'principalStableDebt'
+  | 'stableBorrowRate'
+  | 'stableBorrowLastUpdateTimestamp'
+>;
+*/
+
+export function computeUserReserveData(
```

Comment:
I originally wanted to narrow the types for the more complex functions as well - I didn't find a way though to do this without introducing breaking changes or ugly type hacks.

The problem is, that e.g. `computeUserReserveData` expects the output to be a complete `ReserveData` (because it's spread-ed inside the output). If I would now narrow the input type the output type would alter as well & that might be breaking. Therefore i'd just commented it out and we might want to consider this with more pure functions or generic in a v3.